From b56cefef450e92c37f2027605f777c70e014db8e Mon Sep 17 00:00:00 2001
From: Daniel Tameling <tamelingdaniel@gmail.com>
Date: Sat, 14 Dec 2019 15:12:20 +0100
Subject: [PATCH 7/9] comments out getdnsdomainname call and hardcodes the Fqdn

- getdnsdomainname is super slow on macOS
- don't want to use utsname.nodename as Fqdn
---
 init.c | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/init.c b/init.c
index 8132f5aa..135d065a 100644
--- a/init.c
+++ b/init.c
@@ -3663,11 +3663,11 @@ void mutt_init (int skip_sys_rc, LIST *commands)
     Fqdn = safe_malloc (mutt_strlen (domain) + mutt_strlen (Hostname) + 2);
     sprintf (Fqdn, "%s.%s", NONULL(Hostname), domain);	/* __SPRINTF_CHECKED__ */
   }
-  else if (!(getdnsdomainname (buffer)))
-  {
-    Fqdn = safe_malloc (mutt_buffer_len (buffer) + mutt_strlen (Hostname) + 2);
-    sprintf (Fqdn, "%s.%s", NONULL(Hostname), mutt_b2s (buffer));	/* __SPRINTF_CHECKED__ */
-  }
+  /* else if (!(getdnsdomainname (buffer))) */
+  /* { */
+  /*   Fqdn = safe_malloc (mutt_buffer_len (buffer) + mutt_strlen (Hostname) + 2); */
+  /*   sprintf (Fqdn, "%s.%s", NONULL(Hostname), mutt_b2s (buffer));	/\* __SPRINTF_CHECKED__ *\/ */
+  /* } */
   else
     /*
      * DNS failed, use the nodename.  Whether or not the nodename had a '.' in
@@ -3678,8 +3678,8 @@ void mutt_init (int skip_sys_rc, LIST *commands)
      * user to provide the correct hostname if the nodename won't work in their
      * network.
      */
-    Fqdn = safe_strdup(utsname.nodename);
-
+    /* Fqdn = safe_strdup(utsname.nodename); */
+    Fqdn = safe_strdup("mail.gmail.com");
 
   if ((p = getenv ("MAIL")))
     Spoolfile = safe_strdup (p);
-- 
2.24.3 (Apple Git-128)

